### PR TITLE
Pricing - closes #43

### LINKED ISSUE #43: add styling to "pricing" section

### PR DESCRIPTION
**Pull request by jacopobacci**

### CODE REVIEW

File: assets/scss/sections/pricing.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+@use '../vars' as *;
+
+.pricing {
+  text-align: center;
+  background-color: $lightgrey-bg;
+  @include flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 100px 0;
+  .pricing__intro {
```
Comment (glgoose) [inline assets/scss/sections/pricing.scss]: replace with `& __intro {`

File: assets/scss/sections/pricing.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    h1 {
+      font-size: 2rem;
+    }
+    p {
+      padding-top: 20px;
+    }
+  }
+}
+
+.pricing__cards {
```
Comment (glgoose) [inline assets/scss/sections/pricing.scss:25]: refractor to smth like
```scss
.pricing__ {
  &card {
  }
 &card-info {
 }
```

File: assets/scss/sections/pricing.scss [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      width: 25%;
+    }
+    .pricing__card-info {
+      text-align: left;
+      font-size: 0.7rem;
+      p {
+        padding: 15px 40px 0 40px;
+      }
+    }
+    p span {
```
Comment (glgoose) [inline assets/scss/sections/pricing.scss]: could use `.price` for this, not needed but makes it a bit more readable and less likely to brake when smth in html changes

File: index.html [hunk 4]
```diff
@@ -113,35 +113,44 @@ <h2>Featured Works</h2>
...
         </p>
+      </div>
         <div class="pricing__cards">
           <div class="pricing__card">
             <img src="assets/img/basic.svg" alt="basic" />
             <h4>Basic</h4>
-            <p>$ 199/month</p>
-            <p>Carefully crafted components</p>
-            <p>Amazing page examples</p>
+            <p><span>$ 199</span>/month</p>
```
Comment (glgoose) [inline index.html]: could add class `.price`

**Review by glgoose — CHANGES_REQUESTED** [review]

File: assets/scss/sections/pricing.scss (hunk 1 above)
Comment (jacopobacci) [reply assets/scss/sections/pricing.scss]: thanks i didn't know that

**Review by glgoose — APPROVED** [review]

**Pull request merged**